.github/workflows/tag-on-version-bump.yml

```
@@ -8,6 +8,8 @@ jobs:
   tag_release:
     permissions:
       contents: write
+      # Allow the push to trigger other workflows (e.g. package release)
+      workflow: write
```

actionlint (1.7.7)

unknown permission scope "workflow". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)